gyro.c

```diff
@@ -0,0 +1,63 @@
+#include <stdio.h>
+#include "pico/stdlib.h"
+#include "hardware/i2c.h"
+#include "mpu6050.h"
+
+/*
+ * Gyro Full Scale	/ Sensitivity
+ * 0 => 250			131
+ * 1 => 500			65.5
+ * 2 => 1000			32.8
+ * 3 => 2000 deg/s		16.4 LSB/(deg/s)
+ *
+ * Accel Full Scale	/ Sensitivity
+ * 0 => 2			16384
+ * 1 => 4			8192
+ * 2 => 8			4096
+ * 3 => 16g			2048 LSB/g
+ */
+
+
+int main(){
+
+
```

Comment:
```suggestion

int main(void)
{
```
Let's try to cut down on unnecessary whitespace.